test/Seq.Mail.Tests/SmtpMailAppTests.cs

```diff
@@ -52,4 +53,28 @@ public async Task EventsAreSentAsMessages()
         Assert.Equal("b", message.HtmlBody.Trim());
         Assert.Null(message.TextBody);
     }
+
+    [Fact]
+    public async Task WhenProtocolSecurityIsNoneCertificateValidationIsSkipped()
```

Comment:
Does this test what it says it tests? Looks like it passes an option (`ProtocolSecurity = ProtocolSecurity.None`) and then checks for that option somewhere else (`options.DisableCertificateValidation`) but does that actually check that validation is skipped?

Reply:
I'd lean towards dropping the test if there is no practical way to assert that it skips certificate validation.